openvpn/crypto/cryptoalgs.hpp

```diff
@@ -56,7 +56,9 @@ namespace openvpn {
       AES_128_GCM,
       AES_192_GCM,
       AES_256_GCM,
-
+// ProMIND
```

Comment:
I have no idea what ProMIND is. This comment is rather confusing.

Reply:
ProMIND is me! :-) It simply it a tag to let you maintainers to easily spot the changed. You can removed them all as soon as you are done with your revision.